public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -643,11 +650,14 @@ export const Explorer = ({
                         rows={explorerData.jsonData}
                         rowsAll={explorerData.jsonDataAll}
                         explorerFields={explorerFields}
-                        timeStampField={queryRef.current![SELECTED_TIMESTAMP]}
+                        timeStampField={
+                          !query[FINAL_QUERY].match(PPL_DESCRIBE_INDEX_REGEX)
+                            ? queryRef.current![SELECTED_TIMESTAMP]
+                            : ''
+                        }
                         rawQuery={appBasedRef.current || queryRef.current![RAW_QUERY]}
                         totalHits={
-                          (isDefaultDataSourceType || appLogEvents) &&
-                          query[SELECTED_TIMESTAMP] !== ''
+                          showTimeBasedComponents
```

Comment:
I've leveraged some of the no-timestamp index workflows for this new case, we could get UX signoff if needed but since it uses existing views with no changes to those, I don't think it's entirely necessary.

Reply:
Sure. This is not a blocker, good to reach out to UX for a quick check and sign-off, more of a flow wise.